common/Cargo.toml

```diff
@@ -16,6 +16,7 @@ anyhow = "1.0.26"
 quinn = "0.8.3"
 lazy_static = "1.4.0"
 fxhash = "0.2.1"
+tokio = "1.18.2"
```

Comment:
We should specify the features needed by `common` (probably just `rt`) and disable default features.

Reply:
It looks like `tokio` doesn't have any default features. I believe the necessary features are being added because "common" isn't being built on its own. (I was unaware of https://doc.rust-lang.org/nightly/cargo/reference/features.html#feature-unification).

If I build "common" on its own, I get ``error[E0432]: unresolved import `tokio::sync::mpsc` ``. I can get rid of this error by enabling the "sync" feature, but the "rt" feature is already enabled because of the "quinn" crate, making it a bit difficult to notice that it wasn't enabled when importing the tokio crate.

I'll go ahead and add "sync" and "rt".